plugins/lua/suspendmanager.lua

```diff
@@ -0,0 +1,130 @@
+local _ENV = mkmodule('plugins.suspendmanager')
+
+local utils = require('utils')
+local gui = require('gui')
+local overlay = require('plugins.overlay')
+local widgets = require('gui.widgets')
+
+--- Loop over all the construction jobs
+---@param fn function A function taking a job as argument
+function foreach_construction_job(fn)
+    for _, job in utils.listpairs(df.global.world.jobs.list) do
+        if job.job_type == df.job_type.ConstructBuilding then
+            fn(job)
+        end
+    end
+end
+
+function isKeptSuspended(job)
+    return suspendmanager_isKeptSuspended(job)
+end
+
+-- TODO: check that this is appropriate
+function isBuildingPlanJob(job)
+    --- @type building
+    local building = dfhack.job.getHolder(job)
+    return building and building.mat_type == -1
+end
+
+function runOnce(prevent_blocking)
+    suspendmanager_runOnce(prevent_blocking)
+end
+
+
+-- Overlay Widgets
+StatusOverlay = defclass(StatusOverlay, overlay.OverlayWidget)
+StatusOverlay.ATTRS{
+    desc='Adds information to suspended building panels about why it is suspended.',
+    default_pos={x=-39,y=16},
+    default_enabled=true,
+    viewscreens='dwarfmode/ViewSheets/BUILDING',
+    frame={w=59, h=3},
+    frame_style=gui.MEDIUM_FRAME,
+    frame_background=gui.CLEAR_PEN,
+    default_enabled=true,
+    overlay_only=true,
```

Comment:
Since this is a repositionable widget, `overlay_only` should not be specified

Reply:
Oh, I think this was an unsuccessful experiment that I forgot to revert. Nice catch.